src/utils/helpers.js

```diff
@@ -216,12 +216,17 @@ class Helpers {
  * @returns {Token} Token
  */
   static checkToken(req) {
-    const { authorization } = req.headers;
-    if (!authorization) {
-      throw new ApiError(401, 'Access denied, Token required');
+    const {
```

Comment:
Where is error thrown when a token is not found anywhere?

Reply:
@Baystef. The red patch indicates that the piece of code has been removed.

Reply:
Yes, I know, and that's the piece that throws an error when no token is found, did you move this elsewhere?

Reply:
The function just does a check. In this implementation. It doesn't throw an error

Reply:
@Baystef there's a middleware function named `authenticate` that does that. this function should ideally only check for a token and return that token or false, the middleware now throws an error or call next depending on the case.